src/main/frontend/pipeline-console-view/pipeline-console/main/ConsoleLogStream.tsx

```diff
@@ -50,6 +52,10 @@ export default function ConsoleLogStream(props: ConsoleLogStreamProps) {
     }
   }, [moveToBottom]);
 
+  const consoleLineHeightCallback = useCallback((height: number) => {
+    setConsoleLineHeight(height);
```

Comment:
I can actually sum the height of each ConsoleLine `div` through this callback to accommodate wrapped lines.

Reply:
Unfortunately that approach causes visual artifacts as each ConsoleLine is rendered and the Virtuoso component noticeably resizes. The current solution is good enough for the majority of cases. The only issues are when there are only a few log lines with very different heights due to text wrapping. Using the max line height in the calculation is the best way to handle this case.